packages/roosterjs-editor-plugins/lib/plugins/Paste/excelConverter/convertPastedContentFromExcel.ts

```diff
@@ -24,6 +24,15 @@ export default function convertPastedContentFromExcel(
         moveChildNodes(fragment, doc?.body);
     }
 
+    // For Excel Online
```

Comment:
I did that at first, but since both will be sharing the same 'safeguard' functions of checking for html and chainSanitizerCallback I combined kept them as one.